pkg/common/configs/configs.go

```diff
@@ -26,11 +26,34 @@ import (
 )
 
 const (
-	HealthCheckInterval = "health.checkInterval"
+	// prefixes
+	PrefixEvent  = "event."
+	PrefixHealth = "health."
+
+	HealthCheckInterval = PrefixHealth + "checkInterval"
+
+	// events
+	CMEventRequestsEnabled = PrefixEvent + "requestsEnabled"
+	CMEventTrackingEnabled = PrefixEvent + "trackingEnabled"
```

Comment:
I think we can just use "event.trackingEnabled" and do away with "requestsEnabled". This was for a legacy feature that wasn't ever fully implemented, so there's no need to support it for backwards compatibility.